aten/src/THC/generic/THCTensorMathPointwise.cu

```diff
@@ -68,6 +68,42 @@ IMPLEMENT_CUDA_TENSOR_BASIC_FUNC(  abs, THCNumerics<real>::abs,   Real)
 #undef IMPLEMENT_CUDA_TENSOR_BASIC_FUNC_
 #undef IMPLEMENT_CUDA_TENSOR_BASIC_FUNC
 
+#if defined(THC_REAL_IS_FLOAT) || defined(THC_REAL_IS_DOUBLE)
+
+#if defined(THC_REAL_IS_FLOAT)
+
+struct TensorATan2Op {
+  __device__ __forceinline__ void operator()(float* out, float* a, float* b) {
+    *out = atan2f(*a, *b);
```

Comment:
Nit: wouldn't it be better to add these to THCNumerics? And eventually add support for half as well?